### PR TITLE
updated for windows

### PR DESCRIPTION
**Pull request by geekrelief**
I got this working on Windows 10. Thanks for making this available!

### CODE REVIEW

**Comment by geekrelief** [comment]
Other than the benchmark. It doesn't seem like all the samples are working. I saw your post on Discord of the gravity_well, but the balls just fall downward.  I'm new to Rust, so maybe I have something misconfigured?

File: module/server.cpp [hunk 1]
```diff
@@ -9,9 +9,17 @@
...
 #include "gdnative.h"
 
+#ifdef _WIN32
+#include <windows.h>
+#endif
+
 
 PluggablePhysicsServer::PluggablePhysicsServer() {
+	#ifdef _WIN32
+	ZeroMemory(&this->fn_table, sizeof(this->fn_table));
```
Comment (Demindiro) [inline module/server.cpp:19]: `zeromem` is defined in `core/os/copymem.h` in Godot, so I would expect it to work with MSVC too (and IIRC Godot does compile with MSVC).

**Review by Demindiro — APPROVED** [review]
Thank you for the PR! It looks good to me though I'm curious about one thing.

File: module/server.cpp (hunk 1 above)
Comment (geekrelief) [reply module/server.cpp:19]: Hmm let me double check.  I had an issue with it when I tried before.

Comment (geekrelief) [reply module/server.cpp:19]: Maybe you're running off an older version of godot?  In 3.x `zeromem` doesn't exist. `core/os/copymem.h` doesn't exist.

Comment (geekrelief) [reply module/server.cpp:19]: https://github.com/godotengine/godot/tree/3.x/core/os I guess `zeromem` wouldn't compile on 3.x then. I can't find it in the codebase.  Should we use `memset`?

Comment (Demindiro) [reply module/server.cpp:19]: It definitely does exist: https://github.com/godotengine/godot/blob/3.3.1-stable/core/os/copymem.h

Comment (geekrelief) [reply module/server.cpp:19]: https://github.com/godotengine/godot/pull/48239

Comment (geekrelief) [reply module/server.cpp:19]: They're just macros, so I'll update my patch to just use `memset`.

**Pull request merged**